airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3758,6 +3758,7 @@ components:
         - failed
         - succeeded
         - cancelled
+      nullable: true
```

Comment:
@cgardens had some concerns around changes like this that set `nullable` to `true` on various resources, as he noted in this comment: https://github.com/airbytehq/airbyte/pull/12260#issuecomment-1124513978

@cgardens could you elaborate more on why this is a dangerous change to make?

And @alafanechere could you elaborate more on why you need to set `nullable: true` on these objects for this PR?

Reply:
This PR introduces the use of a model I did not use before `WebBackendConnectionRead`. This model has a `latestJobStatus` attribute referring to the `JobStatus` model and this property is null when no job has run.
The OpenAPI spec does not declare this field as nullable, so the generated client I am using is failing while validating the API response.

Reply:
@alafanechere what is the expected behavior of the API? would you expect it to just not return that field instead of returning the field as null? if so is it reasonable in this case to make that change? i don't want us to add code debt when there is a short fix we could to avoid it.

Reply:
@lmossman this change is dangerous imo because it is covering up unexpected behavior in our API. When a field is not set, we expect it to not be present in the json object. Instead of making that be the case, we're adding noise in our OpenApi config to get around it. Because we started to do this, we have inconsistent behavior in the API. Sometimes we don't return the field and sometimes we return the field with null. It is a slippery slope and we are already sliding.

If there is a reason why changing the server code is very hard to fix, then maybe we make an exception,  but we should be trying to fix the behavior, not changing the OpenApi config. The cases where this happen should generally be trivial to fix.

Reply:
@cgardens thank you for sharing. I totally agree that It's adding noise. I had great directions from @pmossman in another PR about how we can fix it on the server-side. I will try to make the fix myself but it's not the part of the stack I'm the most comfortable with at the moment 😄 I'll keep you updated. 
I made the OpenApi changes in this PR because it was my only way to implement the current feature in a clean way with the generated client. But this PR can wait for the moment.

Reply:
thank you!

Reply:
@cgardens here's a PR to try to fix the problem server-side: https://github.com/airbytehq/airbyte/pull/13370 .